builder-golang/action.yaml

```diff
@@ -0,0 +1,66 @@
+name: "SLSA builder for golang"
+description: "Build a binary for a golang project"
+author: "GOSST Team"
+
+# Documentation: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions.
+
+inputs:
+  go-version:
+    description: "INPUT: Go version to use"
+    required: true
+
+  repo-token:
+    description: "INPUT: GitHub token with read access"
+    required: false
+    default: ${{ github.token }}
+
```

Comment:
fyi: hmm wondering if there's a usecase for taking in the repo to build as an input. It's not entirely clear to me yet, but sticks on as a possibility.

Reply:
can you create a tracking issue?